dtostorage/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>jdi-cucumber-test-generator</artifactId>
+        <groupId>com.epam.jdi</groupId>
+        <version>0.5.0</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>dtostorage</artifactId>
+    <version>0.5.0</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+        </dependency>
+
+        <dependency>
```

Comment:
This dependency is not required. LoginUserDTO has annotation @Component which is a mistake.

Reply:
ValidationErrorsDTO depends on MethodArgumentNotValidException. This can be avoided by moving logic from constructor to GeneralExceptionHandler.